src/front/spv/mod.rs

```diff
@@ -56,7 +56,15 @@ impl Instruction {
         }
     }
 }
-
+/// OpPhi instruction.
+#[derive(Clone, Default, Debug)]
+pub struct PhiInstruction {
+    /// SPIR-V's ID.
+    pub id: u32,
+
+    /// Tuples of (variable, parent).
+    pub variables: Vec<(u32, u32)>,
```

Comment:
same for fields - everything is automatically visible to children, and none of that needs to be visible by IR or siblings